corehq/form_processor/models.py

```diff
@@ -489,7 +491,7 @@ def form_data(self):
         xml = self.get_xml()
         try:
             form_json = convert_xform_to_json(xml)
-        except XMLSyntaxError:
+        except (XMLSyntaxError, ValueError):
```

Comment:
could this have also been:

```
xml = self.get_xml()
if xml is None:  # form XML is missing from blob storage
   return {}
try:
   # old code ....
```

I think that's a bit more explicit in terms of what's going on. I think `ValueError` may also catch something weird like passing an integer into `convert_xform_to_json`